src/main/java/com/sforce/ws/tools/wsdlc.java

```diff
@@ -124,7 +136,7 @@ public static void run(String wsdlUrl, String destJarFilename, String packagePre
         }
     }
 
-    static void run(String[] args) throws Exception {
+    private static void run(String[] args) throws Exception {
```

Comment:
I'm curious, why did you choose to make this private?

Reply:
Because it's called only from this class? Could roll it back, doesn't really matter.

Reply:
Thats reasonable, you don't have to change it then.